lib/flutter_html.dart

```diff
@@ -33,9 +30,6 @@ class Html extends StatefulWidget {
   ///
   /// **onLinkTap** This function is called whenever a link (`<a href>`)
   /// is tapped.
-  /// **customRender** This function allows you to return your own widgets
-  /// for existing or custom HTML tags.
-  /// See [its wiki page](https://github.com/Sub6Resources/flutter_html/wiki/All-About-customRender) for more info.
   ///
   /// **onImageError** This is called whenever an image fails to load or
```

Comment:
onImageError seems to be removed from the class itself.

Reply:
Ah, true. That was intentional. Updated that documentation.

Thanks!